pavelib/assets.py

```diff
@@ -767,10 +768,23 @@ def webpack(options):
     static_root_lms = Env.get_django_setting("STATIC_ROOT", "lms", settings=settings)
     static_root_cms = Env.get_django_setting("STATIC_ROOT", "cms", settings=settings)
     config_path = Env.get_django_setting("WEBPACK_CONFIG_PATH", "lms", settings=settings)
-    environment = u'NODE_ENV={node_env} STATIC_ROOT_LMS={static_root_lms} STATIC_ROOT_CMS={static_root_cms}'.format(
+    additional_node_env_vars_sorted_json = json.dumps(
+        json.loads(
```

Comment:
@nizarmah Why do we need to use `json.loads`?

Reply:
Okay, the content needs to be sorted, because if it isn't, it won't be consistent when dealing with multiple instances.

And since the `get_django_setting` returns a string, we have to `json.loads` it in order to convert it to a dictionary. Then we sort the dictionary by keys, in order to keep it consistent. And then we dump the dictionary into a string.

However, this string needs to be dumped again when we need to use it (which is the comment below).

This change is mainly in order to make the Webpack Content Hashes consistent, check [this commit please](https://github.com/open-craft/edx-platform/pull/271/commits/63581182bef5bc641f5f8a5a1973e753b67334fc).

Reply:
Oh I see. Thanks for the clarification.